### ISSUE TITLE
Handle unusual server exceptions properly

### ISSUE THREAD

**Issue by MorrisJobke** [owncloud/client#3573]
For the PROPFIND I today had following response:

```
<?xml version="1.0" encoding="utf-8"?>
<d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
  <s:exception>GuzzleHttp\Exception\RequestException</s:exception>
  <s:message>cURL error 60: SSL certificate problem: certificate has expired</s:message>
</d:error>
```

This was during adding a new folder and listing the folders on the server. In this case I just got presented an empty dialog window:

![bildschirmfoto von 2015-08-09 10-48-04](https://cloud.githubusercontent.com/assets/245432/9154343/2b9a8a00-3e8a-11e5-88a7-e1f3b489be3e.png)

Can we improve error handling here too?

cc @ogoffart

**Comment by ckamm** [comment]
The patch addresses this particular instance and will be merged once 2.0.0 is branched.

**Comment by ckamm** [comment]
@Dianafg76 Why 'Needs Info'?
